xla/service/gpu/gemm_fusion_autotuner.cc

```diff
@@ -994,25 +994,54 @@ absl::StatusOr<AutotuneResult> Execute(const AutotuneConfig& config,
   VLOG(2) << fusion->name() << ": time with cuBLAS: " << cublas_duration;
   AutotuneResult best;
   best.mutable_failure()->set_kind(AutotuneResult::DISQUALIFIED);
+
+  absl::Duration best_duration_cudnn = absl::InfiniteDuration();
```

Comment:
Why do we need to separately store those best durations? I thought we just store best + all other durations?

Reply:
the proto, in the current state, only stores information about the best backend. the changes we want to introduce, store best time for each of the backends and we can use the information to compare backends for each fusion.

Reply:
That seems redundant if you store *all* the intermediate results? You can always derive best-for-each from there?

Reply:
I don't think we store all intermediate results--not in the proto certianly. Storing best time for each backend would give an overview of the performance of each backend for that particular fusion which is useful for development purposes as well.